fdbserver/tester.actor.cpp

```diff
@@ -1037,6 +1389,8 @@ ACTOR Future<bool> runTest(Database cx,
 		// Run the consistency check workload
 		if (spec.runConsistencyCheck) {
 			try {
+				wait(timeoutError(checkConsistencyUrgentSim(cx, testers, 18000, spec.databasePingDelay, dbInfo),
```

Comment:
is this intentional to run in addition to the `checkConsistency` below? how about running with certain probability?